### ISSUE TITLE
Getting an exception while trying to use BgeSmallEnV15EmbeddingModel to embed documents to elastic store

### ISSUE THREAD

**Issue by debgang** [langchain4j/langchain4j#1524]
**Describe the bug**
Getting an exception while trying to use BgeSmallEnV15EmbeddingModel to embed documents to elastic store

**Log and Stack trace**
Exception in thread "main" java.lang.RuntimeException: java.util.concurrent.ExecutionException: java.lang.RuntimeException: ai.onnxruntime.OrtException: Error code - ORT_RUNTIME_EXCEPTION - message: Non-zero status code returned while running Add node. Name:'/embeddings/Add_1' Status Message: C:\a\_work\1\s\onnxruntime\core/providers/cpu/math/element_wise_ops.h:560 onnxruntime::BroadcastIterator::Append axis == 1 || axis == largest was false. Attempting to broadcast an axis by a dimension other than 1. 512 by 606

	at dev.langchain4j.model.embedding.AbstractInProcessEmbeddingModel.embedAll(AbstractInProcessEmbeddingModel.java:68)
	at dev.langchain4j.model.embedding.EmbeddingModel.embed(EmbeddingModel.java:34)
	at ElasticsearchEmbeddingStoreExample.main(ElasticsearchEmbeddingStoreExample.java:45)

**To Reproduce**

Please find a sample code.The document's are json files

import com.fasterxml.jackson.databind.ObjectMapper;
import dev.langchain4j.data.embedding.Embedding;
import dev.langchain4j.data.segment.TextSegment;
import dev.langchain4j.model.embedding.*;
import dev.langchain4j.model.embedding.bge.small.en.v15.BgeSmallEnV15EmbeddingModel;
import dev.langchain4j.model.embedding.bge.small.en.v15.BgeSmallEnV15QuantizedEmbeddingModel;
import dev.langchain4j.store.embedding.EmbeddingMatch;
import dev.langchain4j.store.embedding.EmbeddingSearchRequest;
import dev.langchain4j.store.embedding.EmbeddingStore;
import dev.langchain4j.store.embedding.elasticsearch.ElasticsearchEmbeddingStore;
import json.JsonParser;
import org.testcontainers.elasticsearch.ElasticsearchContainer;
import prompts.Prompts;
import salesforce.Document;
import salesforce.SalesforceMetadata;

import java.io.IOException;
import java.util.List;
import java.util.Set;

public class ElasticsearchEmbeddingStoreExample {

    public static void main(String[] args) throws InterruptedException, IOException {

        JsonParser parser = new JsonParser();
        List<Document> documents = parser.getDocuments("C:\\workspace\\langchain4j-examples\\elasticsearch-example\\src\\main\\resources\\metadata");
        ObjectMapper mapper = new ObjectMapper();
        try (ElasticsearchContainer elastic = new ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch:8.9.0")
                .withEnv("xpack.security.enabled", "false")) {
            elastic.start();

            EmbeddingStore<TextSegment> embeddingStore = ElasticsearchEmbeddingStore.builder()
                    .serverUrl("http://" + elastic.getHttpHostAddress())
                    .dimension(384)
                    .indexName("metadata-rag")
                    .build();

            //Change the model here for experiment
            EmbeddingModel embeddingModel = new BgeSmallEnV15EmbeddingModel();

            for (Document document: documents) {
                TextSegment segment1 = TextSegment.from(document.toString());
                Embedding embedding1 = embeddingModel.embed(segment1).content();
                embeddingStore.add(embedding1, segment1);
            }

            Thread.sleep(1000); // to be sure that embeddings were persisted
            Set<String> propmtset = Prompts.promptMap.keySet();
            for (String promptKey : propmtset) {
                String prompt = Prompts.promptMap.get(promptKey);
                Embedding queryEmbedding = embeddingModel.embed(prompt).content();
                List<EmbeddingMatch<TextSegment>> relevant_returns = embeddingStore.findRelevant(queryEmbedding, 5);
                for (int i=0;i<relevant_returns.size();i++) {
                    EmbeddingMatch<TextSegment> embeddingMatch = relevant_returns.get(i);
                    System.out.println(embeddingMatch.score());
                    System.out.println(embeddingMatch.embedded().text());

                }

                System.out.println("-------------------------------------------------------------------------------");
            }
        }
    }
}

**Expected behavior**
It should work with BgeSmallEnV15EmbeddingModel. Other embedding models like "AllMiniLmL6V2EmbeddingModel" and "BgeSmallEnEmbeddingModel" are working with same code

**Please complete the following information:**
- LangChain4j version:0.32.0
- LLM(s) used: NA
- Java version: java 8
- Spring Boot version (if applicable): 

**Additional context**

**Comment by langchain4j** [comment]
Hi @MT2012039, thank you for reporting! Could you please provide a json file which fails?

**Comment by debgang** [comment]
Please find it here (https://github.com/MT2012039/elasticsearch-example/tree/main/src/main/resources/metadata)

**Comment by langchain4j** [comment]
@MT2012039 thank you, I've fixed this bug. You can use `0.34.0-SNAPSHOT` version before it will be released in `0.35.0`:

```xml
<repositories>
        <repository>
            <id>snapshot-repo</id>
            <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
            <snapshots>
                <enabled>true</enabled>
            </snapshots>
        </repository>
</repositories>

<dependencies>
        <dependency>
            <groupId>dev.langchain4j</groupId>
            <artifactId>langchain4j-embeddings-bge-small-en-v15</artifactId>
            <version>0.34.0-SNAPSHOT</version>
        </dependency>
</dependencies>
```
